labelbox/schema/project.py

```diff
@@ -1655,6 +1655,73 @@ def __check_data_rows_have_been_processed(
         return response["queryAllDataRowsHaveBeenProcessed"][
             "allDataRowsHaveBeenProcessed"]
 
+    def get_overview(self, with_issues: bool = False) -> Dict:
```

Comment:
the return should be an object of a class, a python `@dataclass`or similar... since you are doing fair number of conversions, a pydantic model might be a good match.